tests/authAPI.spec.js

```diff
@@ -0,0 +1,119 @@
+import { test, expect } from "@playwright/test";
+import { generateUserCredentials, ERRORS } from "../fixtures";
+import { RegisterAPI } from "../pom/modules/api/registerAPI";
+import { LoginAPI } from "../pom/modules/api/loginAPI";
+
+let loginAPI;
+let registerAPI;
+const { username, email, pass, registerdUser, registerdEmail ,invalidEmail} =
+  generateUserCredentials(5);
+let loginEmail = email;
+let loginPassword = pass;
+
+test.describe.configure({ mode: "serial" });
+
+test.describe("Registration API tests", async () => {
+  test.beforeEach("start pom", async ({ page }) => {
+    registerAPI = new RegisterAPI(page);
+  });
+
+  //negative cases
+  test("register with empty username", async () => {
```

Comment:
Mala sugestija vezana za pisanje test case-ova - probaj da stavis expected result u opis:
`Shouldn't be able to register without username`